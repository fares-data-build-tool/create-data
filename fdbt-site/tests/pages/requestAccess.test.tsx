import * as React from 'react';
import { shallow } from 'enzyme';
import RequestAccess from '../../src/pages/requestAccess';

describe('pages', () => {
    describe('requestAccess', () => {
        it('should render correctly', () => {
            const tree = shallow(<RequestAccess />);
            expect(tree).toMatchSnapshot();
        });
    });
});
