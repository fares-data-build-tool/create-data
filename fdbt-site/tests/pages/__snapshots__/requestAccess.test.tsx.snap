// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages requestAccess should render correctly 1`] = `
<Component
  description="Request Access page of the Fares data build tool"
  errors={Array []}
  title="Request Access - Fares data build tool"
>
  <div
    className="govuk-grid-row"
  >
    <div
      className="govuk-grid-column-two-thirds"
    >
      <h1
        className="govuk-heading-l"
        id="request-access-page-heading"
      >
        Request Access
      </h1>
      <p
        className="govuk-body"
      >
        The Fares Data Build Tool is not open to public registration.
      </p>
      <p
        className="govuk-body"
      >
        Please contact
         
        <a
          className="govuk-link"
          href="/contact"
          id="access_link"
        >
          Fares Data Build Support
        </a>
         
        to register.
      </p>
    </div>
    <div
      className="govuk-grid-column-one-third"
    >
      <div>
        <h1
          className="govuk-heading-s"
        >
          Already have an account?
        </h1>
        <a
          className="govuk-link"
          href="/login"
        >
          Sign in
        </a>
      </div>
    </div>
  </div>
</Component>
`;
